addons/account/models/company.py

```diff
@@ -231,6 +231,28 @@ class ResCompany(models.Model):
     # Autopost Wizard
     autopost_bills = fields.Boolean(string='Auto-validate bills', default=True)
 
+    def _get_default_account_price_include(self):
```

Comment:
Would it not be cleaner to have `default=lambda c: c._get_default_account_price_include()` in order to put this method with other methods? (instead of in the middle of fields :vomiting_face: )